app/src/main/java/com/google/jetpackcamera/MainActivity.kt

```diff
@@ -181,11 +181,11 @@ class MainActivity : ComponentActivity() {
                 }
         )
 
-    private fun getStandardMode(): PreviewMode.StandardMode {
-        return PreviewMode.StandardMode { event ->
-            if (event is PreviewViewModel.ImageCaptureEvent.ImageSaved) {
+    private fun getStandardMode(): ExternalCaptureMode.StandardMode {
+        return ExternalCaptureMode.StandardMode { event ->
+            if (event is ExternalCaptureMode.ImageCaptureEvent.ImageSaved) {
                 @Suppress("DEPRECATION")
-                val intent = Intent(android.hardware.Camera.ACTION_NEW_PICTURE)
+                val intent = Intent(Camera.ACTION_NEW_PICTURE)
```

Comment:
nit: you should leave this as fully qualified and remove the import so we don't have a deprecation warning in the import section.